client/recipient_test.go

```diff
@@ -130,6 +131,10 @@ func TestRecipientsCustomWebhook(t *testing.T) {
 			assert.Equal(t, tr.Details.WebhookSecret, r.Details.WebhookSecret)
 			assert.Equal(t, tr.Details.WebhookPayloads, r.Details.WebhookPayloads)
 			assert.Equal(t, tr.Details.WebhookPayloads.TemplateVariables, r.Details.WebhookPayloads.TemplateVariables)
+			require.Len(t, tr.Details.WebhookHeaders, 1)
+			require.Len(t, r.Details.WebhookHeaders, 1)
```

Comment:
nit: this is maybe better written this way
```suggestion
			if assert.Len(t, r.Details.WebhookHeaders, 1) {
```